core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -559,6 +571,20 @@ class TeamPolicy
       : internal_policy(space_, league_size_request, Kokkos::AUTO(),
                         vector_length_request) {}
 
+  TeamPolicy(const typename traits::execution_space& space_,
+             int league_size_request, const Kokkos::AUTO_t&,
+             const Kokkos::AUTO_t&)
+      : internal_policy(space_, league_size_request, Kokkos::AUTO(),
+                        Kokkos::AUTO()) {
+    first_arg = false;
```

Comment:
We removed `first_arg` in a recent pull request.

Reply:
Oh, I get it. I thought the merge would handle it, but the merge (naturally) didn't remove the instances that were in my newly introduced constructors. Slow moment, thanks for the help